glean-core/ffi/src/lib.rs

```diff
@@ -4,6 +4,8 @@
 
 #![allow(clippy::redundant_closure)]
 
+use std::collections::HashMap;
+use std::iter::Iterator;
```

Comment:
Do we need that import?

Reply:
Looks like no.  I wonder why clippy didn't complain...